src/ports/postgres/modules/deep_learning/test/input_data_preprocessor.sql_in

```diff
@@ -341,7 +341,7 @@ SELECT assert (dependent_vartype   = 'text' AND
 -- data_preprocessor_input_batch was created for boolean dependent var,
 -- trying to create validation data using this table for reference, but
 -- passing a non-boolean dep var should error out.
-select assert(trap_error($TRAP$validation_preprocessor_dl(
+select assert(trap_error($TRAP$SELECT validation_preprocessor_dl(
```

Comment:
This trap_error is still failing for the wrong reason i.e. 'the output table `validation_out` already exists'. We need to add the drop table code`DROP TABLE IF EXISTS validation_out, validation_out_summary;` before running the `select validation_preprocessor_dl` statement